src/components/HeaderBlock/HeaderBlock.js

```diff
@@ -209,22 +201,35 @@ class HeaderBlock extends React.Component<Props> {
       customOnBack,
       theme,
       transparent,
+      leftSideFlex,
     } = this.props;
     const colors = getThemeColors(theme);
 
     return (
       <HeaderRow>
-        <LeftItems sideFlex={sideFlex} style={!centerItems.length && !rightItems.length ? { flexGrow: 2 } : {}}>
+        <LeftItems
+          sideFlex={sideFlex || leftSideFlex}
+          style={!centerItems.length && !rightItems.length && !leftSideFlex ? { flexGrow: 2 } : {}}
+        >
           {(leftItems.length || !!noBack)
             ? leftItems.map((item) => this.renderSideItems(item, LEFT))
             : (
-              <BackIcon
-                icon="back"
-                color={transparent ? colors.control : colors.text}
-                onPress={customOnBack ? () => customOnBack() : () => { navigation.goBack(null); }}
-                fontSize={fontSizes.large}
-                horizontalAlign="flex-start"
-              />)
+              <View
+                style={{
+                  marginTop: -20,
+                  marginBottom: -20,
+                }}
+                key="back"
+              >
+                <BackIcon
+                  icon="back"
+                  color={transparent ? colors.control : colors.text}
+                  onPress={customOnBack ? () => customOnBack() : () => { navigation.goBack(null); }}
```

Comment:
or 
```suggestion
                  onPress={customOnBack || () => navigation.goBack(null)}
```
not sure if we support `??` it would be really cool to use it here